modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -49,6 +51,31 @@
   [& args]
   (apply (get-method driver/describe-table :sql-jdbc) args))
 
+(defmethod sql-jdbc.sync/describe-fks-sql :redshift
+  [driver & {:keys [schema-names table-names]}]
+  (sql/format {:select (vec
+                        {:fk_ns.nspname       "fk-table-schema"
+                         :fk_table.relname    "fk-table-name"
+                         :fk_column.attname   "fk-column-name"
+                         :pk_ns.nspname       "pk-table-schema"
+                         :pk_column.attname   "pk-column-name"
+                         :pk_table.relname    "pk-table-name"})
+               :from   [[:pg_constraint :c]]
+               :join   [[:pg_namespace :fk_ns]     [:= :c.connamespace :fk_ns.oid]
+                        [:pg_class     :fk_table]  [:= :c.conrelid :fk_table.oid]
+                        [:pg_attribute :fk_column] [:= :c.conrelid :fk_column.attrelid]
+                        [:pg_class     :pk_table]  [:= :c.confrelid :pk_table.oid]
+                        [:pg_namespace :pk_ns]     [:= :pk_table.relnamespace :pk_ns.oid]
```

Comment:
```suggestion
                        [:pg_namespace :pk_ns]     [:= :pk_table.relnamespace :pk_ns.oid]
                        [:pg_class     :pk_table]  [:= :c.confrelid :pk_table.oid]
```

Make it easier to read since it's the same order with join for fk_*

Reply:
I initially tried this reordering but had to reverse the order again because this join order is needed. But I've made it consistent at least with fk now in [reorder joins](https://github.com/metabase/metabase/pull/38970/commits/4702a7fdd3fbef98f0dcbcc0ca42bef00269d0ad)